django/db/models/base.py

```diff
@@ -460,9 +460,13 @@ class ModelState:
     adding = True
     fields_cache = ModelStateCacheDescriptor()
     related_managers_cache = ModelStateCacheDescriptor()
+    peers = ()
 
     def __getstate__(self):
         state = self.__dict__.copy()
+
+        state.pop("peers", None)
```

Comment:
we should add an assertion for this in some model pickle test, can't quite determine where though.

Reply:
Note to self: I just rebased and removed this line for now (since other changes removed ModelState.__getstate__ altogether). Will need to add a test and make sure this works (it is probably broken right now in the patch).